tests/network/l2_bridge/test_bridge_nic_hot_plug.py

```diff
@@ -448,6 +449,22 @@ def test_vmi_spec_updated_with_hot_plugged_interface(
             interface_name=hot_plugged_interface.name,
         )
 
+    @pytest.mark.polarion("CNV-12223")
+    @pytest.mark.dependency(
+        name="test_no_restart_needed_status_in_vm_after_hot_plug",
+        depends=["test_vmi_spec_updated_with_hot_plugged_interface"],
+    )
+    def test_no_restart_needed_status_in_vm_after_hot_plug(
+        self,
+        running_vm_for_nic_hot_plug,
+    ):
+        with pytest.raises(TimeoutExpiredError):
+            running_vm_for_nic_hot_plug.wait_for_condition(
+                condition="RestartRequired",
+                status=running_vm_for_nic_hot_plug.Condition.Status.TRUE,
+                timeout=TIMEOUT_20SEC,
+            )
```

Comment:
for debugging it will be useful to add an error log (see pytest.raises examples)